src/codeu/chat/server/AuthorityModel.java

```diff
@@ -0,0 +1,23 @@
+/* Original interface by Yuhang Liao, 
+ * annotation and imports by Sarah Abowitz.
+ * 
+ * Hey idk if the new v of this is worth it
```

Comment:
You should delete comments like these - keep them to only what's informative to readers of your code.